src/um/avrt.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2016 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+
+use um::winnt::LONGLONG;
+
+ENUM_NEG!{enum AVRT_PRIORITY {
+    AVRT_PRIORITY_VERYLOW = -2,
```

Comment:
I'd rather this be written as `-2i32 as u32` and just using `ENUM!` instead of creating a new macro.